pdata/ptrace/json_test.go

```diff
@@ -84,7 +84,7 @@ var tracesOTLPFull = func() Traces {
 	sp.SetDroppedLinksCount(1)
 	sp.SetEndTimestamp(pcommon.NewTimestampFromTime(time.Now()))
 	sp.SetParentSpanID(spanID)
-	sp.SetTraceState("state")
+	sp.TraceStateStruct().FromRaw("state")
```

Comment:
Can't we keep having the setter like it was before?

Reply:
The rule of pdata: If it is mutable we don't have a setter. If it is immutable then we have a setter. In this proposal the struct is mutable, so we don't offer a setter (users should change in place).